docs/source/ecf.rst

```diff
@@ -1117,6 +1117,19 @@ This file describes the transit/eclipse and systematics parameters and their pri
       - ``osc_t1`` - This offset determined the start phase of the sinusoidal function.
 
          The time-dependent amplitude and period are defined as follows: ``amp = osc_amp * np.exp(-osc_amp_decay * (time - osc_t0))`` and ``per = osc_per * np.exp(-osc_per_decay * (time - osc_t0))``.  The damped oscillator model is then defined as: ``osc = 1 + amp * np.sin(2 * np.pi * (time - osc_t1) / per)``.  Note that ``osc[time < osc_t0] = 1``.
+         
+   - Star Spot Parameters
+      - ``spotstari`` - The stellar inclination in degrees.
```

Comment:
Does this parameter only matter if `spotrot` >> T_14? It would probably be helpful to add a small word of guidance to users on how they might set priors on this parameter or when they might choose to fix it to 90 degrees or something.

Reply:
only matters when the star is rotating, so that the spots move appropriately. Will add text about this